### DISCUSSION TITLE
How to generating nested modules in JavaScript from Rust nested modules

### DISCUSSION THREAD

**Discussion by tyxk8160** [wasm-bindgen/wasm-bindgen#3551] [Q&A]
### Summary
How  to generating nested modules in JavaScript from Rust nested modules

### Additional Details
Here is my use case
```rust
pub mod lunnar {
  #[derive(Debug)]
  #[wasm_bindgen]
  pub struct YearCalender {
      inner: Rlunnar::YearCalender,
  }
}
```
Here are wasm-pack generate:
```typescript

export class YearCalender {
  // 
}

```
I hope  JS calss `YearCalender `  under module `lunnar `

**Comment by ranile** [comment]
As far as I'm aware, you can't generate nested modules. You can, however, easily write a JS wrapper around generated code
